src/components/ExpandedNode/ExpandedNode.tsx

```diff
@@ -83,6 +87,18 @@ const ExpandedNodeField: FC<ExpandedNodeFieldProps> = ({ title, description }) =
   );
 };
 
+function renderStatusFields(documentation: DocumentationResource): ReactElement[] {
+  const returnValue: ReactElement[] = [];
+
+  const status = documentation.status;
+
+  const rawDate = documentation.resource?.meta?.lastUpdated;
+  const date = rawDate && new Date(rawDate).toLocaleString('en-us');
+  returnValue.push(<ExpandedNodeField key="Status" title={status} description={date} />);
```

Comment:
Are we expecting to have other ExpandedNodeFields being returned by this function?

Reply:
Not sure. Originally I had it as two fields "status" and "status date" but then the design had it more like this. I can turn it back into a single item instead of an array if we confirm that's what we want for the design

Reply:
Changed it to return just the single field not an array